CHANGELOG.md

```diff
@@ -94,3 +94,32 @@ the following form: YYYY.0M.0D.
   - accountExists() method
   - main() method with example bank account use
   - implemented a constructor for bank account
+
+## [2024.10.17]
```

Comment:
Looks good. I would just make sure the newest changes are at the top of the file.